pkg/agent/endpoints/sds/handler.go

```diff
@@ -54,6 +55,10 @@ type Handler struct {
 	}
 }
 
+func (h *Handler) DeltaSecrets(discovery_v2.SecretDiscoveryService_DeltaSecretsServer) error {
```

Comment:
Please define this below NewHandler with the rest of the functions that have a *Handler receiver

Reply:
sure